packages/server-core/src/sequelize.ts

```diff
@@ -33,7 +33,9 @@ export default (app: Application): void => {
       promiseReject = reject
     })
 
-    app.setup = async function (...args: any) {
+    app.setup = async function (...args) {
+      const testModeForceRefresh = app.get('testModeForceRefresh')
```

Comment:
rather than setting this manually, we can rely on `appConfig.testEnabled`, such that it is automatically applied every time `createFeathersExpressApp` is run

Reply:
I tried to do this initially but encountered some problems. DB is also reseeded before tests (in `npm run pretest`) which requires [`process.exit(0)` here](https://github.com/EtherealEngine/etherealengine/blob/dev/packages/server-core/src/sequelize.ts#L104). Using `appConfig.testEnabled` will have some problems.

Reply:
Ahh I see. Let's use the appConfig instead of feathers internal state. We can add a new configuration option with the desired functionality.

Reply:
Added this change!